cookbooks/bach_repository/attributes/default.rb

```diff
@@ -0,0 +1,43 @@
+#
+# "bins" contains all binaries and repositories, plus some builds
+# "src" contains some builds
+#
+# The decision on whether to build in src_directory vs bins_directory
+# was made by the old build_bins.sh.  For now, we are just replicating
+# the behavior of the old script.
+#
+default[:bach][:repository][:bins_directory] = '/home/vagrant/chef-bcpc/bins'
+default[:bach][:repository][:src_directory] = '/home/vagrant/chef-bcpc/src'
+
+# What is this for?
```

Comment:
This is just a version identifier for the repo, it could (and perhaps should) be the BACH release number.